### PR TITLE
Port CSharpEventHookup

### PR DESCRIPTION
**Pull request by rchande**
Tag @dotnet/roslyn-ide for review

### CODE REVIEW

File: src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpEventHookUp.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        Console.CancelKeyPress$$
+    }
+}");
+            VisualStudio.Editor.SendKeys(" +=", VirtualKey.Tab);
+            VisualStudio.Workspace.WaitForAsyncOperations(FeatureAttribute.EventHookup);
```
Comment (dpoeschl) [inline src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpEventHookUp.cs]: This wait shouldn't be necessary. The async token is to track from "+=" to QuickInfo, not until Tab. The Tab should block. I claim.

File: src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpEventHookUp.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        Console.CancelKeyPress$$
+    }
+}");
+            VisualStudio.Editor.SendKeys(" +=");
+            VisualStudio.Workspace.WaitForAsyncOperations(FeatureAttribute.EventHookup);
```
Comment (dpoeschl) [inline src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpEventHookUp.cs]: (but this wait is good)

File: src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpEventHookUp.cs (hunk 1 above)
Comment (dpoeschl) [inline src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpEventHookUp.cs]: You really ended up needing this? I'm super surprised.

**Comment by rchande** [comment]
I’m not quite sure—the tests failed after I removed it, so I’m putting it back to see what happens.

**Comment by rchande** [comment]
retest windows_debug_vs-integration_prtest

**Pull request closed without merging**